recipes/hictk/build.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+export CONAN_NON_INTERACTIVE=1
+
+export CMAKE_BUILD_PARALLEL_LEVEL=${CPU_COUNT}
+export CTEST_PARALLEL_LEVEL=${CPU_COUNT}
+
+if [[ ${DEBUG_C} == yes ]]; then
+  CMAKE_BUILD_TYPE=Debug
+else
+  CMAKE_BUILD_TYPE=Release
+fi
+
+scratch=$(mktemp -d)
+export CONAN_HOME="$scratch/conan"
+
+# shellcheck disable=SC2064
+trap "rm -rf '$scratch'" EXIT
+
+# Conan doesn't detect compiler name and version when using cc/c++
+TMPBIN="$scratch/bin"
+mkdir "$TMPBIN"
+
+declare -a CMAKE_PLATFORM_FLAGS
+if [[ ${HOST} =~ .*darwin.* ]]; then
+  export MACOSX_DEPLOYMENT_TARGET=10.15  # Required to use std::filesystem
+  CMAKE_PLATFORM_FLAGS+=(-DCMAKE_OSX_SYSROOT="${CONDA_BUILD_SYSROOT}")
+  conan_profile='clang'
+else
+  CMAKE_PLATFORM_FLAGS+=(-DCMAKE_TOOLCHAIN_FILE="${RECIPE_DIR}/cross-linux.cmake")
+
+  ln -sf "$CC" "$TMPBIN/gcc"
+  ln -sf "$CXX" "$TMPBIN/g++"
+
+  export PATH="$TMPBIN:$PATH"
+  export CC="$TMPBIN/gcc"
+  export CXX="$TMPBIN/g++"
+  conan_profile='gcc'
+fi
+
+# Remember to update these profiles when bioconda's compiler toolchains are updated
+mkdir -p "$CONAN_HOME/profiles/"
+ln -s "${RECIPE_DIR}/conan_profiles/$conan_profile" "$CONAN_HOME/profiles/$conan_profile"
+
+conan install conanfile.txt \
+       --build="*" \
+       --build="b2/*" \
+       -pr:b "$conan_profile" \
+       -pr:h "$conan_profile" \
+       --output-folder=build/
```

Comment:
The dependencies which are installed with conan should be listed in the run section of the meta.yaml instead of being installed with conan.

Reply:
Thanks for your comments.
I've updated the conanfile.txt to only list header-only dependencies.
Some of these dependencies are not available on conda-forge and I prefer to use a package manager like Conan to download and configure them instead of doing it manually.